matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/query/QueryStringValueProcessor.kt

```diff
@@ -16,28 +16,28 @@
 
 package org.matrix.android.sdk.internal.query
 
-import org.matrix.android.sdk.api.query.QueryStringValue
 import io.realm.Case
 import io.realm.RealmObject
 import io.realm.RealmQuery
+import org.matrix.android.sdk.api.query.QueryStringValue
 import timber.log.Timber
 
 fun <T : RealmObject> RealmQuery<T>.process(field: String, queryStringValue: QueryStringValue): RealmQuery<T> {
     when (queryStringValue) {
         is QueryStringValue.NoCondition -> Timber.v("No condition to process")
-        is QueryStringValue.IsNotNull   -> isNotNull(field)
-        is QueryStringValue.IsNull      -> isNull(field)
-        is QueryStringValue.IsEmpty     -> isEmpty(field)
-        is QueryStringValue.IsNotEmpty  -> isNotEmpty(field)
-        is QueryStringValue.Equals      -> equalTo(field, queryStringValue.string, queryStringValue.case.toRealmCase())
-        is QueryStringValue.Contains    -> contains(field, queryStringValue.string, queryStringValue.case.toRealmCase())
```

Comment:
new formatting rules?

Reply:
We need to switch yes, but maybe in a separate PR will revert.
This formatting is better for visibility, but it's annoying because adding a line could change all block if the new line is longer (and it's annoying for diff). Also there is a bug on Android Studio that do not respect this rule when doing a quick format on the file, hence this change?